lldb/packages/Python/lldbsuite/test/decorators.py

```diff
@@ -1,6 +1,6 @@
 # System modules
 from functools import wraps
-from pkg_resources import packaging
+from packaging.version import parse
```

Comment:
```suggestion
from packaging import version
```

`parse` sounds too generic to be imported by base name alone